main.lua

```diff
@@ -0,0 +1,104 @@
+imgxKaizer=200
+imgyKaizer=0
+
+imgxC1 = 0
+imgyC1 = 100
+
+imgxC2 = 0
+imgyC2 = 200
+
+imgxC3 = 0
+imgyC3 = 300

```

Comment:
there is a logical grouping for these variables, and width and height, consider putting them into a "car" object and "kaiser" object.

Reply:
Tom fixed by adding attributes into "Car" and "Kaizer" objects when implementing tables.